docs/manual/composites.md

```diff
@@ -18,10 +18,12 @@ With composite nodes, you can craft unique and dynamic Behavior Trees for your g
 
 ## Restarting a composite
 
-If a parent node restarts a child node, it means that the parent node will start the child node from scratch the next time it is ticked. This means that any progress made by the child node will be reset, and it will start its execution from the beginning.
+When a parent node restarts, it means the whole composite node begins its evaluation again from the beginning. This does not interrupt the child nodes.
 
-## Ticking again a composite
+## Interrupting child nodes
+
+A sequence may interrupt any `RUNNING` child node in case a child returns a `FAILURE`. The `interrupt` method will be called on all child nodes of the currently evaluated child node.
```

Comment:
> The `interrupt` method will be called on all child nodes of the currently evaluated child node.

The term _child nodes_ typically denotes only the direct children, i.e. the first-order descendants. For clarity, I'd suggest using the term _descendant nodes_ to denote all descending nodes and say `... will be called on all descendant nodes`.

Also, instead of `the currently evaluated child node`, I'd say `the interrupted child node` just to make it clear which node we are referring to.